packages/atomic/src/components/insight/atomic-insight-interface/e2e/page-object.ts

```diff
@@ -0,0 +1,110 @@
+import {BasePageObject} from '@/playwright-utils/base-page-object';
+import {Page} from '@playwright/test';
+
+export class InsightInterfacePageObject extends BasePageObject<'atomic-insight-interface'> {
+  constructor(page: Page) {
+    super(page, 'atomic-insight-interface');
+  }
+
+  get insightInterface() {
+    return this.page.getByTestId('root-interface');
+  }
+
+  get insightTabs() {
+    return this.page.locator('atomic-insight-tab');
+  }
+
+  get insightQuerySummary() {
+    return this.page.locator('atomic-insight-query-summary');
+  }
+
+  get insightSearchBox() {
+    return this.page.locator('atomic-insight-search-box');
+  }
+
+  get insightRefineToggle() {
+    return this.page.locator('atomic-insight-refine-toggle');
+  }
+
+  get insightUserActionsToggle() {
+    return this.page.locator('atomic-insight-user-actions-toggle');
+  }
+
+  get insightPager() {
+    return this.page.locator('atomic-insight-pager');
+  }
+
+  get insightPagerButtons() {
+    return this.insightPager.getByRole('radio');
+  }
+
+  get insightResults() {
+    return this.page.locator('atomic-insight-result');
+  }
+
+  get quickviewModal() {
+    return this.page.locator('atomic-quickview-modal');
+  }
+
+  get atomicQuickviewModal() {
+    return this.page.locator('.atomic-quickview-modal');
+  }
```

Comment:
you have duplicate locators here

Reply:
humm good point lol although if I remember correctly they were not actually the same target of the locator 😂

Reply:
Guess the first one was not used anywhere